### PR TITLE
Add @group legacy on CRUDControllerTest

### PR DESCRIPTION
**Pull request by Soullivaneuh**
### Subject

This will avoid deprecation notice because of the following PR: #3865

cc @greg0ire

### CODE REVIEW

File: Tests/Controller/CRUDControllerTest.php [hunk 1]
```diff
@@ -38,6 +38,9 @@
 /**
  * Test for CRUDController.
  *
+ * NEXT_MAJOR: Remove this legacy group.
+ * @group legacy
```
Comment (greg0ire) [inline Tests/Controller/CRUDControllerTest.php]: Does this really need to be put on the whole class?

File: Tests/Controller/CRUDControllerTest.php [hunk 2]
```diff
@@ -38,6 +38,9 @@
 /**
  * Test for CRUDController.
  *
+ * NEXT_MAJOR: Remove this legacy group.
```
Comment (OskarStark) [inline Tests/Controller/CRUDControllerTest.php]: why not using `@todo` ?

Comment (Soullivaneuh) [inline Tests/Controller/CRUDControllerTest.php]: https://github.com/sonata-project/dev-kit/blob/master/project/CONTRIBUTING.md#the-base-branch

Comment (OskarStark) [inline Tests/Controller/CRUDControllerTest.php]: didn't notice that, thank you!

**Comment by OskarStark** [comment]
LGTM 👍

**Pull request merged**